dolphinscheduler-task-plugin/dolphinscheduler-task-sql/src/main/java/org/apache/dolphinscheduler/plugin/task/sql/SqlTask.java

```diff
@@ -365,13 +379,31 @@ private String executeUpdate(Connection connection, List<SqlBinds> statementsBin
         int result = 0;
         for (SqlBinds sqlBind : statementsBinds) {
             try (PreparedStatement statement = prepareStatementAndBind(connection, sqlBind)) {
+
+                sqlLogListener(statement);
+
                 result = statement.executeUpdate();
                 logger.info("{} statement execute update result: {}, for sql: {}", handlerType, result, sqlBind.getSql());
             }
         }
         return String.valueOf(result);
     }
 
+    /**
+     * sql log listener
+     *
+     * @param statement statement
+     * @throws SQLException sqlException
+     */
+    private void sqlLogListener(PreparedStatement statement) throws SQLException {
+        if (DbType.HIVE.name().equals(sqlParameters.getType())) {
+            logger.info("The current sql task type is HIVE, will start a demon thread to resolve the appIds");
```

Comment:
```suggestion
            logger.info("The current sql task type is HIVE, will start a daemon thread to resolve the appIds");
```

Reply:
ok, I will modify it according to your suggestions. Can you help me merge the code after modification

Reply:
ok, I will modify it according to your suggestions. Can you help me merge the code after modification